docs/rules/no-deprecated-functions.md

```diff
@@ -9,10 +9,20 @@ of majors, eventually they are removed completely.
 ## Rule details
```

Comment:
Should this mention that it respects/varies based on jest version?

Reply:
You mean something like [this](https://github.com/jest-community/eslint-plugin-jest/pull/564/files#diff-8471e3f5cb1f63cdf65debe2f509b2ddR12)?

(But could be improved by mentioning the actual setting, or at least linking back to that section in the readme)

Reply:
Linking back or something yeah, so people can be explicit